src/core/scene/a-scene.js

```diff
@@ -1,24 +1,26 @@
 /* global Promise, customElements, screen, CustomEvent */
-var initMetaTags = require('./metaTags').inject;
-var initWakelock = require('./wakelock');
-var loadingScreen = require('./loadingScreen');
-var scenes = require('./scenes');
-var systems = require('../system').systems;
-var components = require('../component').components;
-var THREE = require('../../lib/three');
-var utils = require('../../utils/');
-var warn = utils.debug('core:a-scene:warn');
+import THREE from '../../lib/three.js';
+import { inject as initMetaTags } from './metaTags.js';
+import { initWakelock } from './wakelock.js';
+import * as loadingScreen from './loadingScreen.js';
+import scenes from './scenes.js';
+import { systems } from '../system.js';
+import { components } from '../component.js';
+import * as utils from '../../utils/index.js';
+
 // Require after.
-var AEntity = require('../a-entity').AEntity;
-var ANode = require('../a-node').ANode;
-var initPostMessageAPI = require('./postMessage');
+import { AEntity } from '../a-entity.js';
+import { ANode } from '../a-node.js';
+import { initPostMessageAPI } from './postMessage.js';
+
+import '../../utils/ios-orientationchange-blank-bug.js';
```

Comment:
Conditional logic is moved into this class. So while it's always "loaded", it's only applied for iOS. This is simply due to the fact that `require()` is synchronous, but `import()` isn't. Which would then either require a top-level await, or an awkwardly placed `.then()`